src/report/business-reports/payables.scm

```diff
@@ -50,7 +51,7 @@
       "w" (N_ "The payable account you wish to examine.") 
       #f #f (list ACCT-TYPE-PAYABLE)))
```

Comment:
Better place to point this out:
Here you can add your own internal option (using gnc:make-internal-option). You could for example call it "receivable" and give it a default value #f to get as close as possible to your original intent.